lldb-eval/context.h

```diff
@@ -119,7 +119,8 @@ class Context : public ParserContext {
   }
   lldb::SBExecutionContext GetExecutionContext() const { return ctx_; }
 
-  void SetContextVars(std::unordered_map<std::string, TypeSP> context_vars);
+  void SetContextVars(std::unordered_map<std::string, Value> context_vars);
```

Comment:
The evaluator doesn't use `Context` now, so there should never be `context_vars` if the parser only operates with types, right?
We've discussed the optimization of "using the values directly if they're available", but I think for the first approach it's better to drop it for simplicity.

Reply:
Changed `Context` to operate with context arguments only.